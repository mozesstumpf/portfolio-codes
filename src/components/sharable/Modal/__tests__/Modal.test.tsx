import React from 'react';
import { renderer, AllTheProviders } from 'tests/utilities';
import { Modal } from 'components/sharable';

it('should render correctly', () => {
	const tree = renderer
		.create(
			<AllTheProviders>
				<Modal
					zIndex={9999}
					maxWidth={'1000px'}
					disableScroll={false}
					isCloseOutsideClick={true}
					// eslint-disable-next-line @typescript-eslint/no-empty-function
					closeModal={() => {}}
				>
					<div>Modal content</div>
				</Modal>
			</AllTheProviders>,
		)
		.toJSON();

	expect(tree).toMatchSnapshot();
});
