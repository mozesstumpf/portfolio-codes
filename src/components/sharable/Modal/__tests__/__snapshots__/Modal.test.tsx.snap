// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="Modal__Container-sc-hqhzu9-0 kaeSxP"
>
  <div
    className="Modal__Content-sc-hqhzu9-1 VGEZs"
  >
    <div>
      Modal content
    </div>
  </div>
</div>
`;
